### PR TITLE
[WFLY-4382] Functionality to allow symbolic links

### PR DESCRIPTION
**Pull request by jakubpycha**

### CODE REVIEW

**Comment by jakubpycha** [comment]
Yes, he can.

File: undertow/src/main/java/org/wildfly/extension/undertow/handlers/FileHandler.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public HttpHandler createHandler(final OperationContext context, ModelNode model
         String path = PATH.resolveModelAttribute(context, model).asString();
         boolean directoryListing = DIRECTORY_LISTING.resolveModelAttribute(context, model).asBoolean();
         UndertowLogger.ROOT_LOGGER.creatingFileHandler(path);
-        FileResourceManager resourceManager = new FileResourceManager(new File(path), 1024 * 1024);
+        FileResourceManager resourceManager = new FileResourceManager(new File(path), 1024 * 1024, true, new String[0]);
```
Comment (ctomc) [inline undertow/src/main/java/org/wildfly/extension/undertow/handlers/FileHandler.java]: this should be enabled by configuration option not by default.
similarly as it was done in https://issues.jboss.org/browse/WFLY-4254 for web applications

Comment (jakubpycha) [inline undertow/src/main/java/org/wildfly/extension/undertow/handlers/FileHandler.java]: I agree and i'll fix it.

**Comment by ctomc** [comment]
I've send bit more complete PR to address also this here: #7440

**Comment by jakubpycha** [comment]
That are bad news for me, because i hava also make it in my local git so as it do you have in your PR. This issue was recommended me as good first issue and i had it as work on school open source project.

**Comment by ctomc** [comment]
@jakubpycha sorry about that, i did it as part of other changes as it required few more things.
the safePath param is bit tricky to define in mgmt model.
I will try to find something simple for your as first task if you want.

**Pull request closed without merging**